docs/compute/python.md

```diff
@@ -1 +1,151 @@
+    <figcaption>Installing library using search</figcaption>
+  </figure>
+</p>
+
+###Install from package file
+
+Install a python package by uploading the corresponding package file and selecting it in the file browser.
+
+Currently supported formats include:
+
+* .whl
+* .egg
+* requirements.txt
+* environment.yml
+
+<p align="center">
+  <figure>
+    <img src="../../assets/images/python/install_dep.gif" alt="Installing library from file">
+    <figcaption>Installing library from file</figcaption>
+  </figure>
+</p>
+
+###Install from .git repository
+
+To install from a git repository simply provide the repository URL. The URL you should provide is the same as you would enter on the command line using `pip install git+{repo_url}`.
+In the case of a private git repository, also select whether it is a GitHub or GitLab repository and the preconfigured access token for the repository.
+
+**Note**: If you are installing from a git repository which is not GitHub or GitLab simply supply the access token in the URL. Keep in mind that in this case the access token may be visible in logs for other users in the same project to see.
+
+<p align="center">
+  <figure>
+    <img src="../../assets/images/python/install_git.gif" alt="Installing library from git repo">
+    <figcaption>Installing library from git repo</figcaption>
+  </figure>
+</p>
+
+##Uninstalling libraries
+
+To uninstall a library, find the library in the list of installed libararies tab and click the trash icon to remove the library.
+
+<p align="center">
+  <figure>
+    <img src="../../assets/images/python/uninstall.png" alt="Uninstall a library">
+    <figcaption>Uninstall a library</figcaption>
+  </figure>
+</p>
+
+##Debugging the environment
+
+After each installation or uninstall of a library, the environment is analyzed to detect libraries that may not work properly. In order to do so we use the ``pip check`` tool, which is able to identify missing dependencies or if a dependency is installed with the incorrect version.
+The alert will automatically show if such an issue was found.
+
+
+<p align="center">
+  <figure>
+    <img src="../../assets/images/python/conflicts.png" alt="Conflicts detected in environment">
+    <figcaption>Conflicts detected in environment</figcaption>
+  </figure>
+</p>
+
+##Recreating environment
+
+Sometimes it may be desirable to recreate the environment to start from the default project environment. In order to do that, first click `Remove env`.
+
+<p align="center">
+  <figure>
+    <img src="../../assets/images/python/remove_env.png" alt="Remove environment">
+    <figcaption>Remove environment</figcaption>
+  </figure>
+</p>
+
+After removing the environment, simply recreate it by clicking `Create Environment`.
+
+<p align="center">
+  <figure>
+    <img src="../../assets/images/python/create_env.png" alt="Create environment">
+    <figcaption>Create environment</figcaption>
+  </figure>
+</p>
+
+##Exporting an environment
```

Comment:
Do we support importing a yml file in the new UI? Also I would move this section before the `Remove environment`.

Reply:
I moved the Export before the Remove environment. There is no option to import an environment.yml and create an environment from that at the moment, it also came to my knowledge recently. You can however install an environment.yml or requirements.txt on top of the default environment right now.